### PR TITLE
Fixed syntax error in the doc

### PR DESCRIPTION
**Pull request by AkshayBz**
Fixed syntax error which was blocking preview generation. 
Error was due to migration form md to rst.

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -58,20 +58,20 @@ Step 2: Enable the bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enable the bundle in the kernel:
+ 
+.. code-block:: php
```
Comment (stof) [inline Resources/doc/index.rst]: the Symfony documentation conventions are to use the ``::`` shortcut at the end of the previous paragraph when we have a paragraph ending with a ``:`` followed by a code-block in the default language (i.e. PHP): http://sphinx-doc.org/rest.html#source-code 
Can you change it ?

Comment (AkshayBz) [inline Resources/doc/index.rst]: Correct me if I am wrong, I think what you are saying is without syntax highlighting.
What I used is correct syntax for a code block: http://sphinx-doc.org/markup/code.html

Comment (AkshayBz) [inline Resources/doc/index.rst]: Oops. Got what you mean.
Sorry, having a bad day :stuck_out_tongue: 
~BadExcuseIKnow

**Comment by AkshayBz** [comment]
Just had a look at all the documentations, 'configuration-block' directive is used a lot but it isn't rendered on Github.
That's why I was having a hard time today figuring out the documentation itself :stuck_out_tongue_winking_eye:

**Comment by sstok** [comment]
:+1:

**Comment by stof** [comment]
@AkshayBz this is because the goal of converting the doc to rST is to be able to have it rendered on symfony.com, which is in progress. See the discussion in https://github.com/FriendsOfSymfony/FOSUserBundle/issues/1901

**Comment by stof** [comment]
thanks @AkshayBz. I rebased your work to fix the doc in the 1.3.x branch too (and then merged the 1.3.x branch into master). this is why this PR got marked as closed rather than merged. You still get attribution for the change in a7da47f1dc4ddd0e004336635de828fc89e4aeb7 of course

**Comment by AkshayBz** [comment]
Great thanks! :)

My question is that even though the documentation will be rendered on symfony.com, shouldn't there be compatibility with Github and other native rST viewer too?
Linters show it as a syntax error and it's pretty serious considering the fact that the whole block disappears while rendering and the user doesn't even know that it's gone..

**Comment by stof** [comment]
@AkshayBz rST supports the concept of directives, and this is precisely what is used here. If linters mark the custom directive as a syntax error, there is 2 possibilities:

- either the linter needs to be aware of custom directives being used => you can give it the code for the directive as it is open-source: https://github.com/fabpot/sphinx-php/blob/master/sensio/sphinx/configurationblock.py
- or the linter does not support the concept of custom directives => it does not respect the rST spec and should be fixed

And regarding the fact that the github rendering does not render the custom directive (as it does not know about it), this has been the case for the past 4 years in the symfony-docs repo, and it has been fine for people. I think the confusion is mainly due to the switch. but all documentation links for the bundle have now been switched to point to the symfony.com website.

**Pull request closed without merging**